payment-method-basic-card/payment-request-canmakepayment-method.https.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<!-- Copyright © 2017 World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
+<meta charset="utf-8">
+<title>Payment Method Basic Card: Tests that basic card is a supported method</title>
+<link rel="help" href="https://w3c.github.io/browser-payment-api/#show-method">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+const basicCard = Object.freeze({ supportedMethods: "basic-card", data: {} });
+const defaultMethods = Object.freeze([basicCard]);
+const defaultDetails = Object.freeze({
+  total: {
+    label: "Total",
+    amount: {
+      currency: "USD",
+      value: "1.00",
+    },
+  },
+});
+
+const notSupportedMethod = Object.freeze({
+  supportedMethods: "this-is-not-supported",
+});
+
+promise_test(async t => {
+  const request = new PaymentRequest([notSupportedMethod], defaultDetails);
+  assert_false(
+    await request.canMakePayment(),
+    `canMakePaymentPromise should be false`
+  );
+}, `Must return false when the PMI is not supported at by the user agent.`);
```

Comment:
This and the other tests fail as such in Chrome:
promise_test: Unhandled rejection with value: object "TypeError: Failed to construct 'PaymentRequest': The provided value cannot be converted to a sequence."

Chrome bug, or test bug?

Reply:
Chrome bug in this case.

Reply:
OK, thanks for poking at it with a test!